modules/ROOT/pages/bat-command-reference.adoc

```diff
@@ -107,21 +98,61 @@ Validates the file or folder. Also works with `--config` to validate specific co
 
+** `environment ls|list|switch`
++
+The two options `ls` and `list` list the environments that are available for the specified profile. If not profile is specified, the default profile is used.
++
+The `switch` option lets you use one of the listed environments. Add the name of the environment to the end of the command, as in this example that uses the environment `Sandbox`:
++
+----
+bat environment switch Sandbox
+----
+
+** `execution ls|list`
++
+Lists the monitors that are currently running for the specified profile. If no profile is specified, the default profile is used. The monitors listed are all of those in the current environment and for the current profile that were written and started through the BAT CLI, and any that were created online in Anypoint Platform.
++
+The output looks like this:
++
+----
+Last executions:
+
+	daily-retail-test:1.0.1                  PASSED     10 seconds ago
+	internal-run-api-check:1.0.0             PASSED     11 minutes ago
+----
+
 ** `get_token`
 +
 Returns the token for accessing Anypoint Platform in the credentials file. Requires that you first run `bat login` to generate a profile.
 
+** grant
++
+Grants Anypoint Functional Monitoring permission to access a shared secret that is stored in Anypoint Secrets Manager. You must grant Anypoint Functional Monitoring permission to access any shared secret that you use in a test or to access a reporter.
++
+Run the `bat grant` command before configuring a test or a reporter to use a shared secret. The aliases that you create in the command are what you specify in tests and configurations for reporters.
++
+----
+bat grant -g=MySecretGroupName -s=MyAlias1:MySecretName1,MyAlias2:MySecretName2,…,MyAliasN:MySecretNameN [--forceUpdate]
+----
++
+-g:: Specify the name of the group in Anypoint Secrets Manager in which the shared secret is stored.
+-s:: Specify an alias, a colon, and then the name of the shared secret. The alias can be any combination of alphanumeric characters. The name of the shared secret is the name that identifies the shared secret in the group. If you want to use a single command to grant the BAT CLI permission to use more than shared secret that is from the same group, separate each `alias:name` pair with a comma.
++The secret must be created in a secrets group that is in the same environment that you are using in the BAT CLI. To find out which environment you are currently using in the BAT CLI, run the command `bat whoami`. In the output, there is the ID for the environment. Run the command `bat environment ls` to list the environments that you have access to. Match the ID from the `bat whoami` command with one of the environments listed. If you need to switch to the environment that your secrets group is in, run the command `bat environment switch name`, where `name` is the name of the environment.
```

Comment:
Here is the text that I thought did not belong in Step 1a of the task in a previous topic. 
Can we refer to this text there instead of duplicating it?

Reply:
```suggestion
+The secret must be created in a secrets group that is in the same environment that you are using in the BAT CLI. To find out which environment you are currently using in the BAT CLI, run the command `bat whoami`. The output contains the ID for the environment. Run the command `bat environment ls` to list the environments that you have access to. Match the ID from the `bat whoami` command with one of the environments listed. If you need to switch to the environment that your secrets group is in, run the command bat environment switch name, where name is the name of the environment.
```